tests/api/commands/test_flags.py

```diff
@@ -28,12 +28,12 @@
 from typer.testing import CliRunner
 
 
-def test_format(runner, snapshot):
+def test_format(runner, custom_snapshot):
```

Comment:
I changed all snapshots to `custom_snapshot`. In next changes I will try to nail down exact places where custom comparator is neccessary.